ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -35,6 +35,8 @@ using namespace skia::textlayout;
 std::queue<rnsKey> inputQueue;
```

Comment:
Can all this global be changed to as class member

Reply:
explained offline